File: projekt2/src/proj/java/model/plant/Grass.java
package proj.java.model.plant;

import java.awt.Color;

import proj.java.model.Plant;
import proj.java.model.World;

public class Grass extends Plant{

    private static final int GRASS_STRENGTH = 0;

    @Override
    protected void spread(int newRow, int newCol, World world) {
        world.setOrganism(newRow, newCol, new Grass());
    }

    @Override
    public int getStrength() {
        return GRASS_STRENGTH;
    }

    @Override
    public Color getColor() {
        return Color.GREEN;
    }
}

File: projekt3/Grass.py
from Plant import *

class Grass(Plant):
    __GRASS_STRENGTH__ = 0
    __COLOR__ = "Green"
    
    def __init__(self, row, col, world):
        super().__init__(row, col, self.__GRASS_STRENGTH__,world)
    
    def getColor(self):
        return self.__COLOR__
    
    def getName(self):
        return "Grass"

    def spread(self, newRow, newCol):
        self._world_.setOrganism(newRow, newCol, Grass(newRow , newCol, self._world_))
